ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -1,16 +1,49 @@
-export function generateMetadata(): string {
+import { LoopPermissions } from './loop-permissions';
+
+function isPermissionUndefined(permission: string[]): boolean {
+  if (permission.length === 0 || 
```

Comment:
One tiny quick thing - I think you'd want `if (permission === undefined || permission.length === 0) { ...`, just because the `.length` access will fail if `permission` is `undefined`, so checking that first should guard it

Reply:
sweet! Need the team to figure out if this is something we would want from an Loop Author perspective.

Reply:
I was actually going to ask if there's a guarantee that that object will always be there and have a particular shape. Can we know that permissions will be there and will be an object?